src/assets/scripts/client/navigationLibrary/ProcedureModel.js

```diff
@@ -128,6 +128,17 @@ export default class ProcedureModel {
          */
         this._procedureType = '';
 
+        /**
+         * The initial climb clearance (only for SIDs)
+         *
+         * @for ProcedureModel
```

Comment:
`@for` is only needed for method doc blocks

Reply:
done